_pages/en_US/ctrtransfer.txt

```diff
@@ -111,7 +111,7 @@ If script found no user tickets told you to skip this section, then skip this se
 
 #### Section V - Remove CTRTransfer image
 
-1. Launch GodMode9 by holding (Start) during boot
+1. Launch GodMode9. Press and hold (Start), and while holding (Start), power on your device
```

Comment:
I don't see the reason to switch up the wording here. I'd understand if you're trying to avoid repetition but you're still repeating the same text, just in a different order.